softdevice_controller/include/sdc_hci_vs.h

```diff
@@ -205,6 +207,13 @@ typedef __PACKED_STRUCT
     sdc_hci_vs_zephyr_static_address_t addresses[];
 } sdc_hci_cmd_vs_zephyr_read_static_addresses_return_t;
 
+/** @brief Zephyr Read Chip Temperature return parameter(s). */
+typedef __PACKED_STRUCT
+{
+    /** @brief The measured temperature in degrees Celsius. */
+    int8_t temps;
```

Comment:
`temp` or `temps`?

Reply:
yeah, it's temps in the zephyr spec I don't know why

Reply:
I would propose to just name it temp. `temps` looks like a typo